contracts/Oracles/ChainlinkPriceOracle.sol

```diff
@@ -40,7 +40,7 @@ contract ChainlinkPriceOracle is BasePriceOracle, IChainlinkPriceOracle {
 
     // --- Functions ---
 
-    function getPriceOracleResponse() external view override returns (PriceOracleResponse memory) {
+    function getPriceOracleResponse() external override returns (PriceOracleResponse memory) {
```

Comment:
why removing view?

Reply:
Because for `rETH`, I must use `(uint256 ethUsdPrice,) = priceFeedETH.fetchPrice();`, which is not view.

Reply:
rek